pkg/controller/storageprofile-controller_test.go

```diff
@@ -45,11 +45,11 @@ var (
 
 var _ = Describe("Storage profile controller reconcile loop", func() {
 
-	It("Should return error if storage profile can not be found", func() {
```

Comment:
not sure if possible or if its not easy if its worth working on it but maybe should check if can add a test which returns error which is different then not found and see that it does reque.

Reply:
Yeah that is a challenge, not sure how I could simulate this scenario